src/plugins/apimon/apimon.cpp

```diff
@@ -309,6 +310,49 @@ static event_response_t usermode_hook_cb(drakvuf_t drakvuf, drakvuf_trap_info* i
     return VMI_EVENT_RESPONSE_NONE;
 }
 
+static void print_addresses(drakvuf_t drakvuf, apimon *plugin, const dll_view_t *dll, const std::vector<hook_target_view_t> *targets)
```

Comment:
As `targets` is not an optional parameter it's type should be `const std::vector<hook_target_view_t>&`.